workflow-bridge-lambda/src/index.ts

```diff
@@ -62,6 +86,24 @@ const getPinboardById =
     return { ...data.externalData, ...data };
   };
 
+const getPinboardsByPaths = async (identifiers: number[] | string[]) => {
```

Comment:
when can identifiers be `number[]`? I only see `string[]`

Reply:
ohhh good spot thanks, think this was a hangover from when the method was more generic and support being passed numerous workflow ids (which are numbers) - corrected now